runtime/doc/diagnostic.txt

```diff
@@ -475,55 +475,55 @@ config({opts}, {namespace})                          *vim.diagnostic.config()*
                        severities are displayed before lower severities (e.g.
                        ERROR is displayed before WARN). Options:
                        • reverse: (boolean) Reverse sort order
-      • {namespace}  (integer|nil) Update the options for the given namespace.
+      • {namespace}  (`integer?`) Update the options for the given namespace.
```

Comment:
Did we consider dropping the parens? Especially since we have backticks now.

Reply:
No but we could do. I kind of like the parens since it makes it clear it is separate from the description, but I don't feel super strong about it. The backticks will be concealed but will add highlighting.